geode-core/src/distributedTest/java/org/apache/geode/cache30/DistributedAckRegionCCEDUnitTest.java

```diff
@@ -147,129 +146,118 @@ public void testConcurrentOpWithGII() {
+      CCRegion = (LocalRegion) f1.create(name);
+      return CCRegion.getDistributionManager().getDistributionManagerId();
+    });
 
     AsyncInvocation partialCreate =
-        vm2.invokeAsync(new SerializableCallable<Object>("create region with stall") {
-          @Override
-          public Object call() {
-            final GemFireCacheImpl cache = (GemFireCacheImpl) getCache();
-            RegionFactory f = cache.createRegionFactory(getRegionAttributes());
-            InitialImageOperation.VMOTION_DURING_GII = true;
-            // this will stall region creation at the point of asking for an initial image
-            VMotionObserverHolder.setInstance(new VMotionObserver() {
-              @Override
-              public void vMotionBeforeCQRegistration() {}
-
-              @Override
-              public void vMotionBeforeRegisterInterest() {}
-
-              @Override
-              public void vMotionDuringGII(Set recipientSet, LocalRegion region) {
-                InitialImageOperation.VMOTION_DURING_GII = false;
-                final InitializationLevel oldLevel =
-                    LocalRegion.setThreadInitLevelRequirement(BEFORE_INITIAL_IMAGE);
-                LocalRegion ccregion = (LocalRegion) cache.getInternalRegionByPath("/" + name);
-                try {
-                  // wait for the update op (sent below from vm1) to arrive, then allow the GII to
-                  // happen
-                  while (!ccregion.isDestroyed() && ccregion.getRegionEntry(key) == null) {
-                    try {
-                      Thread.sleep(1000);
-                    } catch (InterruptedException e) {
-                      return;
-                    }
+        vm2.invokeAsync("create region with stall", () -> {
+
+          final GemFireCacheImpl cache = (GemFireCacheImpl) getCache();
+          RegionFactory f = cache.createRegionFactory(getRegionAttributes());
+          InitialImageOperation.VMOTION_DURING_GII = true;
+          // this will stall region creation at the point of asking for an initial image
+          VMotionObserverHolder.setInstance(new VMotionObserver() {
+            @Override
+            public void vMotionBeforeCQRegistration() {}
+
+            @Override
+            public void vMotionBeforeRegisterInterest() {}
+
+            @Override
+            public void vMotionDuringGII(Set recipientSet, LocalRegion region) {
+              InitialImageOperation.VMOTION_DURING_GII = false;
+              final InitializationLevel oldLevel =
+                  LocalRegion.setThreadInitLevelRequirement(BEFORE_INITIAL_IMAGE);
+              LocalRegion ccregion = (LocalRegion) cache.getInternalRegionByPath("/" + name);
+              try {
+                // wait for the update op (sent below from vm1) to arrive, then allow the GII to
+                // happen
+                while (!ccregion.isDestroyed() && ccregion.getRegionEntry(key) == null) {
```

Comment:
I recommend changing any sort of loop containing a sleep into an `await` call:
```
await().ignoreException(RegionDestroyedException.class).until(() ->
    ccregion.isDestroyed() && ccregion.getRegionEntry(key) != null);
```

Reply:
done